### PR TITLE
Increase external media link input width, and increase top value of globe icon

### LINKED ISSUE #32249: Image block: Replace Current media URL field too small
## Description
When editing an image's URL using the **Replace** tool bar option the **Current media URL** field is far too narrow to be of much use.

## Step-by-step reproduction instructions

Please list the steps needed to reproduce the bug. For example:
1. Drag an image from another browser window.
2. Choose Replace from the image block toolbar
3. Notice that the field is far too narrow to be usable. 

## Expected behaviour
The Current media URL field should at least try to be wide enough to display half of the URL.

## Actual behaviour
You can only see approximately 16 characters. 

## Screenshots or screen recording (optional)
In this screen capture the image URL is `https://herbmiller.me/wp-content/uploads/2021/05/image-1024x513.png`

image

## WordPress information
- WordPress version: 5.7.2
- Gutenberg version: 10.6.2 
- Are all plugins except Gutenberg deactivated? Yes 
- Are you using a default theme (e.g. Twenty Twenty-One)? TT1 blocks 

## Device information
- Device: Desktop 
- Operating system: Windows 10
- Browser: Chrome Version 90.0.4430.212 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by tjcafferkey**
Fixes https://github.com/WordPress/gutenberg/issues/32249

## Description

* Increase the width of the external media input element
* Increase top value of globe icon

## How has this been tested?

* Create a new post or edit an existing post
* Add "Image" block and select "Insert from URL" option
* Enter an image URL
* Now select the "Replace" button. Observe the correct alignment of the globe icon.
* Click "Edit" to edit the URL. Box should expand to make the input more usable.

## Screenshots 

<table>
<tr>
	<td>Before</td>
	<td>After</td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
</table>

## Types of changes

Bug fix

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

**Comment by tjcafferkey** [comment]
Failing E2E tests appear unrelated? Is there any way I can restart these?

**Comment by tjcafferkey** [comment]
>Failing E2E tests appear unrelated? Is there any way I can restart these?

cc @getdave

**Comment by getdave** [comment]
Yep. Click on failing test details link. Then click restart tests.

File: packages/block-editor/src/components/media-replace-flow/style.scss [hunk 1]
```diff
@@ -44,6 +44,14 @@
 			padding: 0;
 		}
 
+		.block-editor-link-control__search-item-icon {
+			top: 1em;
+		}
```
Comment (getdave) [inline packages/block-editor/src/components/media-replace-flow/style.scss:49]: Screen Shot 2021-07-30 at 09 28 06

Not sure this is necessary any longer as I was unable to replicate. I merged a PR recently which sorted out all this globe icon alignment so hopefully that's done it.

**Review by getdave — CHANGES_REQUESTED** [review]
Thanks for looking into fix this. Being able to see the full URL when in "edit" mode is very important.

Unfortunately whilst this works well on larger screens, on smaller viewports there appears to be a severe bug whereby the modal visually "vibrates".

https://user-images.githubusercontent.com/444434/127626375-20a3f551-2045-4250-b594-18b2bb0a2831.mp4

This appears to be due to some circular logic in the JS which causes a `style` attribute to be repeatedly set/unset.

Screen Capture on 2021-07-30 at 09-37-47

If we can find a way to resolve that then we'll be in a good place. I think maybe it's the `min-width` you've set on the `.block-editor-link-control__search-input` which you'll need to apply (using a progressive enhancement approach) _only_ on larger viewports.

**Comment by getdave** [comment]
@tjcafferkey Anything I can do to help this along?

**Comment by tjcafferkey** [comment]
Sorry @getdave I have been AFK for a few weeks. I will be picking this back up. Thanks for checking in though.

**Comment by getdave** [comment]
@tjcafferkey How's this PR coming along? Can I help at all?

**Pull request closed without merging**